# Description: This pacth adds a missing link against Kross.
#  Without this patch the package will FTBFS with binutils-gold.
# Forwarded: not-needed
# Author: José Manuel Santamaría Lema <panfaust@gmail.com>
# 
--- subtitlecomposer-0.5.3.orig/src/main/CMakeLists.txt
+++ subtitlecomposer-0.5.3/src/main/CMakeLists.txt
@@ -46,6 +46,7 @@
 	${KDE4_KDEUI_LIBS}
 	${KDE4_KHTML_LIBS}
 	${KDE4_KFILE_LIBS}
+	${KDE4_KROSSCORE_LIBS}
 	${KDE4_KROSSUI_LIBS}
 	${common_LIBS}
 	${config_LIBS}
